src/main/groovy/org/ajoberstar/grgit/auth/AuthConfig.groovy

```diff
@@ -97,11 +99,13 @@ class AuthConfig {
     static final String FORCE_OPTION = 'org.ajoberstar.grgit.auth.force'
     static final String USERNAME_OPTION = 'org.ajoberstar.grgit.auth.username'
     static final String PASSWORD_OPTION = 'org.ajoberstar.grgit.auth.password'
+	static final String PASSPHRASE_OPTION = 'org.ajoberstar.grgit.auth.passphrase'
     static final String SSH_PRIVATE_KEY_OPTION = 'org.ajoberstar.grgit.auth.ssh.private'
```

Comment:
Looking back at this name, it's not that great.

Maybe a new alternative should be added and use of this one should print a deprecation message?

`org.ajoberstar.grgit.auth.ssh.keypath`